packages/codepen-loader/jest.config.js

```diff
@@ -6,6 +6,7 @@ const path = require('path');
 module.exports = {
   preset: 'ts-jest',
   testRegex: '/__tests__/.*\\.test\\.ts$',
+  reporters: [path.resolve(__dirname, '../../scripts/jest/jest-reporter.js')],
```

Comment:
What's the point of this reporter? this now makes a package depend on the build scripts but through relative path ... I actually removed this dependency before.

Reply:
It makes it not send errors to stderr when there's not actually an error. Though that may be less important if we're not using rush and if we don't end up needing the failOnStderr option in build tasks.